uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerAuthenticatorUpdatedVersionImpl.java

```diff
@@ -47,7 +46,7 @@
 public class BrokerAuthenticatorUpdatedVersionImpl extends BrokerMicrosoftAuthenticator {
 
     private static final String TAG = BrokerAuthenticatorUpdatedVersionImpl.class.getSimpleName();
-    public static boolean shouldUseDeviceSettingsPage = true;
+    public static boolean shouldUseDeviceSettingsPage = false;
```

Comment:
Do we have some validation to make sure this doesn't affect other test cases?

Reply:
I ran multiple tests locally with this change succesfully.